e2e/e2e_test.go

```diff
@@ -0,0 +1,172 @@
+package e2e
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	whbar "github.com/limechain/hedera-eth-bridge-validator/app/clients/ethereum/contracts/whbar"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	common "github.com/ethereum/go-ethereum/common"
+	"github.com/hashgraph/hedera-sdk-go"
+	ethclient "github.com/limechain/hedera-eth-bridge-validator/app/clients/ethereum"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	validatorproto "github.com/limechain/hedera-eth-bridge-validator/proto"
+	"google.golang.org/protobuf/proto"
+)
+
+func Test_E2E(t *testing.T) {
+	configuration := config.LoadTestConfig()
+
+	memo := "0x7cFae2deF15dF86CfdA9f2d25A361f1123F42eDD1126221237211"
+	whbarReceiverAddress := common.HexToAddress("0x7cFae2deF15dF86CfdA9f2d25A361f1123F42eDD")
+
+	hBarAmount := 0.0001
+	validatorsCount := 3
+	ethSignaturesCollected := 0
+	ethTransMsgCollected := 0
+	ethTransactionHash := ""
+
+	whbarContractAddress := common.HexToAddress(configuration.Hedera.Eth.WhbarContractAddress)
+	acc, _ := hedera.AccountIDFromString(configuration.Hedera.Client.Operator.AccountId)
+	receiving, _ := hedera.AccountIDFromString(configuration.Hedera.Watcher.CryptoTransfer.Accounts[0].Id)
+	topicID, _ := hedera.TopicIDFromString(configuration.Hedera.Watcher.ConsensusMessage.Topics[0].Id)
+	ethConfig := configuration.Hedera.Eth
+
+	accID, _ := hedera.AccountIDFromString(configuration.Hedera.Client.Operator.AccountId)
+	pK, _ := hedera.PrivateKeyFromString(configuration.Hedera.Client.Operator.PrivateKey)
+
+	client := initClient(accID, pK)
+	ethClient := ethclient.NewEthereumClient(ethConfig)
+	whbarInstance, err := whbar.NewWhbar(whbarContractAddress, ethClient.Client)
+
+	// Get the wrapped hbar balance of the receiver before the transfer
+	whbarBalanceBefore, err := whbarInstance.BalanceOf(&bind.CallOpts{}, whbarReceiverAddress)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("WHBAR balance before transaction: [%s]\n", whbarBalanceBefore)
+
+	// Get custodian hbar balance before transfer
+	recieverBalance, err := hedera.NewAccountBalanceQuery().
+		SetAccountID(receiving).
+		Execute(client)
+
+	fmt.Println(fmt.Sprintf(`HBAR custodian balance before transaction: [%d]`, recieverBalance.Hbars.AsTinybar()))
+
+	// Get the transaction receipt to verify the transaction was executed
+	transactionResponse, err := sendTransactionToCustodialAccount(acc, receiving, memo, hBarAmount, client)
+	transactionReceipt, err := transactionResponse.GetReceipt(client)
+
+	if err != nil {
+		fmt.Println(fmt.Sprintf(`Transaction unsuccessful, Error: [%s]`, err))
+		t.Fatal(err)
+	} else {
+		fmt.Println(fmt.Sprintf(`Successfully sent HBAR to custodian adress, Status: [%s]`, transactionReceipt.Status))
```

Comment:
I think you can use this without 'else' block

Reply:
fixed :)